src/queens/utils/classifier.py

```diff
@@ -45,9 +43,7 @@ class Classifier:
 
     is_active = False
 
-    def __init__(
-        self, n_params: int, classifier_obj: skactiveml.classifier._wrapper.SklearnClassifier
-    ) -> None:
+    def __init__(self, n_params: int, classifier_obj: "SklearnClassifier") -> None:
```

Comment:
if we import `from __future__ import annotations`

we can do:
```suggestion
    def __init__(self, n_params: int, classifier_obj: SklearnClassifier) -> None:
```